scripts/ssl.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+myDir=$(dirname "$0")
+source "$myDir/config.sh"
+
+currentTime=$(date +%s)
+
+printf "SSL Certificates:\n"
+for domain in $ssl_domains; do
+    delimiter="\t"
+    certTime=$(openssl s_client -connect ${domain}:443 < /dev/null 2>/dev/null | openssl x509 -noout -enddate 2>/dev/null | cut -d= -f2)
+    certLineTime=$(date -d "${certTime}" +"%a %b %d %Y")
+
+    certTimestamp=$(date -d "${certTime}" +%s)
+    if [ "${certTimestamp}" -ge "${currentTime}" ]; then
```

Comment:
This test breaks when certTime is empty, which can happen if openssl cannot connect to the host.